src/OrbitGl/CallstackThreadBar.cpp

```diff
@@ -226,22 +226,23 @@ bool CallstackThreadBar::IsEmpty() const {
   return callstack_count == 0;
 }
 
-CallstackThreadBar::ModuleAndFunctionNameToFormat CallstackThreadBar::SafeGetModuleAndFunctionNames(
-    const CallstackInfo& callstack, size_t frame_index) const {
+CallstackThreadBar::UnformattedModuleAndFunctionName
+CallstackThreadBar::SafeGetModuleAndFunctionName(const CallstackInfo& callstack,
+                                                 size_t frame_index) const {
   ORBIT_CHECK(capture_data_ != nullptr);
   if (frame_index >= callstack.frames().size()) {
-    ModuleAndFunctionNameToFormat module_and_function_name_to_format;
-    module_and_function_name_to_format.module_name =
-        orbit_client_data::kUnknownFunctionOrModuleName;
-    module_and_function_name_to_format.module_is_unknown = true;
-    module_and_function_name_to_format.function_name =
-        orbit_client_data::kUnknownFunctionOrModuleName;
-    module_and_function_name_to_format.function_is_unknown = true;
-    return module_and_function_name_to_format;
+    static const UnformattedModuleAndFunctionName kFrameIndexTooLargeModuleAndFunctionName = [] {
```

Comment:
I was actually wondering to reduce boilerplate code here and below. So maybe a constructor/factory method is better. But all just nit thinkings.

Reply:
I was thinking of something like `UnformattedModuleAndFunctionName::CreateUnknownModuleAndFunctionName()` And then in the case where we at least know the address, override the function name.

Reply:
Having both is redundant. The alternative would be to have this constant in `UnformattedModuleAndFunctionName`, but since I don't think `UnformattedModuleAndFunctionName` should know about `orbit_client_data::kUnknownFunctionOrModuleName` I'm leaving it here.